src/Marten/MartenRegistry.cs

```diff
@@ -293,10 +293,10 @@ public DocumentMappingExpression<T> FullTextIndex(Action<FullTextIndex> configur
                 return this;
             }
 
-            public DocumentMappingExpression<T> ForeignKey(Expression<Func<T, object>> expression, string schemaName, string tableName, string columnName,
+            public DocumentMappingExpression<T> ForeignKey(Expression<Func<T, object>> expression, string tableName, string columnName, string schemaName,
```

Comment:
```suggestion
            public DocumentMappingExpression<T> ForeignKey(Expression<Func<T, object>> expression, string schemaName, string tableName, string columnName,
```
Please keep previous signature as @mysticmind suggested